<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleConfig</class>
 <widget class="QWidget" name="StyleConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>204</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="formAlignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelApplicationStyle">
     <property name="text">
      <string>Application style:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QComboBox" name="comboStyle">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbConfigStyle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Configure Application Style...</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="configure">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelShowIcons">
     <property name="text">
      <string>Show icons:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelToolbarIcons">
     <property name="text">
      <string>Main toolbar label:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboToolbarIcons">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Beside icons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Below icons</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelSecondaryToolbarIcons">
     <property name="text">
      <string>Secondary toolbar label:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="comboSecondaryToolbarIcons">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Text only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Beside icons</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Below icons</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="cbIconsOnButtons">
     <property name="text">
      <string>On buttons</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="cbIconsInMenus">
     <property name="text">
      <string>In menus</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
